experiments/AMIP/modular/components/land/bucket_init.jl

```diff
@@ -16,14 +16,16 @@ import ClimaLSM.Bucket:
     surface_albedo,
     snow_precipitation
 
-using ClimaLSM: make_ode_function, initialize, obtain_surface_space, make_set_initial_aux_state
+using ClimaLSM:
+    make_ode_function, initialize, obtain_surface_space, make_set_initial_aux_state, surface_evaporative_scaling
 
 """
-    BucketSimulation{P, Y, D, I}
+    BucketSimulation{M, P, Y, D, I}
 
 The bucket model simulation object.
 """
-struct BucketSimulation{P, Y, D, I}
+struct BucketSimulation{M, P, Y, D, I}
+    model::M
     params::P
```

Comment:
if you dont want to store params, that is fine - it should be stored in `model.parameters`

Reply:
I'm planning to make this change in a separate PR so this one doesn't become too large - see issue [here](https://github.com/CliMA/ClimaCoupler.jl/issues/273)

Reply:
perfect!